ui/tests/acceptance/secrets/backend/aws/aws-configuration-test.js

```diff
@@ -63,6 +63,21 @@ module('Acceptance | aws | configuration', function (hooks) {
     await runCmd(`delete sys/mounts/${path}`);
   });
 
+  test('it should redirect if old url is entered', async function (assert) {
+    const path = `aws-${this.uid}`;
+    await enablePage.enable('aws', path);
+    await click(SES.configTab);
+    await visit(`/vault/settings/secrets/configure/${path}`);
```

Comment:
Since there used to be another `/section` piece of this URL path, I wonder if we should also have a test and case handling for a url like `/vault/settings/secrets/configure/${path}/leases` in case someone has a really old URL bookmarked

Reply:
Good idea!